lessons/Data-Science/measures-of-center/langs/en-us/index.adoc

```diff
@@ -403,6 +335,56 @@ _Kids need to see the shape of the data first_, otherwise mean, median and mode
 Looking at the shape of the data puts ground under your feet. Otherwise, you're too focused on numbers, and measures of center don't actually mean anything.
 }
 
+
+== Histograms and Measures of Center
+
+=== Overview
+
+Students combine apply their knowledge histograms, measures of center, and shape.
+
+=== Launch
+
+Can you compute and interpret different measures of center from a histogram? Let's try it.
+
+@QandA{
+The histogram below shows the number of children per home in Broadmoore neighborhood.
+
+@image{images/number-children.png, 350}
+
+@Q{What is the median number of children per home?}
+@A{1. Students may mistakenly attempt to find the midpoint of the values on the horizontal axis, indicating that they connect median with “middle”, but misunderstand _what_ middle value to find. In this case, all of the raw data is available in the histogram.
```

Comment:
The `1.` here by itself looks like a typo. I actually started this comment to tell you it was a mistake!🤣

Make the sentence longer, with "The median is 1."